tests/Substrait.Lineage.Tests/LineageExtractorVisitorTest.cs

```diff
@@ -0,0 +1,19 @@
+using Substrait.Protobuf;
+
+namespace Substrait.Lineage.Tests
+{
+  [TestClass]
+  public class LineageExtractorVisitorTest
+  {
+    [TestMethod]
+    public void TestMethod1()
+    {
+      Plan plan;
+      using var planPb = File.OpenRead(@"TestData\tpcds\Q02.pb");
```

Comment:
Consider `Path.Combine` so that test works irrespective of os directory separator.